csrankings-j.csv

```diff
@@ -1678,7 +1678,7 @@ John K. Antonio,University of Oklahoma,http://www.ou.edu/coe/cs/people/antonio,9
 John K. Bennett,University of Colorado Boulder,https://www.colorado.edu/cs/users/jbk,Yp8S-_QAAAAJ
 John K. Ousterhout,Stanford University,http://web.stanford.edu/~ouster,nDH-AqwAAAAJ
 John K. Tsotsos,York University,http://www.cse.yorku.ca/~tsotsos/Tsotsos/Home.html,Ic6ZXhUAAAAJ
-John Kenneth Salisbury Jr.,Stanford University,https://med.stanford.edu/profiles/john-salisbury,FmrDR68AAAAJ
+John Kenneth Salisbury Jr.,Stanford University,https://profiles.stanford.edu/john-salisbury?releaseVersion=10.8.0,FmrDR68AAAAJ

```

Comment:
This faculty member should be removed (emeritus and non-tenure track).

Reply:
Thank you for your catch